src/ccutil/ccutil.cpp

```diff
@@ -57,27 +57,30 @@ void CCUtil::main_setup(const std::string &argv0, const std::string &basename) {
 #if defined(_WIN32)
   } else if (datadir.empty() || _access(datadir.c_str(), 0) != 0) {
     /* Look for tessdata in directory of executable. */
-    char path[_MAX_PATH];
+    char path[_MAX_PATH] = {0};
```

Comment:
Why is this change needed?

Reply:
It's not strictly necessary, just found it slightly cleaner this way.

Reply:
It's unnecessary, so let's keep the old code.